packages/snaps-utils/src/snaps.test.ts

```diff
@@ -14,9 +15,36 @@ import {
   assertIsValidSnapId,
   verifyRequestedSnapPermissions,
   stripSnapPrefix,
+  isSnapId,
 } from './snaps';
 import { uri, WALLET_SNAP_PERMISSION_KEY } from './types';
 
+describe('isSnapId', () => {
+  it.each(['npm:@metamask/test-snap-bip44', 'local:http://localhost:8000'])(
+    'returns `true` for "%s"',
+    () => {
+      expect(isSnapId('npm:@metamask/test-snap-bip44')).toBe(true);
```

Comment:
Seems like this is unintentionally using a static value?

Reply:
It is. 😅 Fixed.